modules/aws-http4s/src/smithy4s/aws/http4s/package.scala

```diff
@@ -27,7 +27,7 @@ package object http4s {
       private[this] val service: smithy4s.Service[Alg]
   ) {
 
-    def awsClient[F[_]: Temporal](
+    def awsClient[F[_]: Async](
```

Comment:
this is not binary compatible but we're not making a strong promise at this point, moreover I'm pretty confident it's unlikely to break any user code .